src/share/index.js

```diff
@@ -0,0 +1,21 @@
+import fs from '../lib/fs';
+import Filer from '../../node_modules/filer/dist/filer';
+const sh = new fs.Shell();
+
+import { formatIndex } from '../lib/html-formatter';
+import registerRoute from './routes';
+
+export default class {
+    init(workbox) {
+        registerRoute(workbox, this);
+    }
+
+    async torrent() {
```

Comment:
What is this doing?  Why not just send them to `index.html`? without using the service worker?

Reply:
I was originally building the index page through `html-formatter`, I don't need it anymore so I'll remove it